cheroot/ssl/pyopenssl.pyi

```diff
@@ -32,14 +32,20 @@ class pyOpenSSLAdapter(Adapter):
         certificate_chain: _t.Any | None = ...,
         ciphers: _t.Any | None = ...,
         *,
-        private_key_password: str | bytes | None = ...,
+        private_key_password: str
+        | bytes
+        | _t.Callable[[], bytes | str]
+        | None = ...,
```

Comment:
It may be a good idea to define a new type or a type alias for this (except for the `None` bit I suppose).

Reply:
Okay, disregard this, then. We can revisit when the annotations move from stubs to native syntax.

Reply:
Okay, disregard this, then. We can revisit when the annotations move from stubs to native syntax.